implants/lib/eldritch/src/sys/dll_reflect_impl.rs

```diff
@@ -20,27 +20,27 @@ use {
     },
 };
 
-#[cfg(all(host_family = "windows", target_os = "windows"))]
+#[cfg(all(target_family = "windows", target_os = "windows"))]
 macro_rules! win_target {
     () => {
         r"x86_64-pc-windows-msvc"
     };
 }
-#[cfg(all(host_family = "unix", target_os = "windows"))]
+#[cfg(all(target_family = "unix", target_os = "windows"))]
 macro_rules! win_target {
     () => {
         r"x86_64-pc-windows-gnu"
     };
 }
 
-#[cfg(all(host_family = "unix", target_os = "windows"))]
+#[cfg(all(target_family = "unix", target_os = "windows"))]
 macro_rules! sep {
     () => {
         "/"
     };
 }
 
-#[cfg(host_family = "windows")]
+#[cfg(target_family = "windows")]
```

Comment:
Unfortunately target_family can't replace host_family.

host_family is a custom variable we set in build.rs that defines the OS we're building on. It's important since rusts build.rs doesn't offer an OS agnostic way to do file paths in macros so this is the the work around 🫠